bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -224,12 +224,40 @@ public Collection<WorkItem> prRun(Path scratchPath) {
         // First determine if the current state of the PR has already been checked
         var seedPath = bot.seedStorage().orElse(scratchPath.resolve("seeds"));
         var hostedRepositoryPool = new HostedRepositoryPool(seedPath);
-
-        var census = CensusInstance.createCensusInstance(hostedRepositoryPool, bot.censusRepo(), bot.censusRef(), scratchPath.resolve("census"), pr,
-                                           bot.confOverrideRepository().orElse(null), bot.confOverrideName(), bot.confOverrideRef()).orElseThrow();
+        CensusInstance census = null;
         var comments = pr.comments();
         var allReviews = pr.reviews();
         var labels = new HashSet<>(pr.labelNames());
+        try {
+            census = CensusInstance.createCensusInstance(hostedRepositoryPool, bot.censusRepo(), bot.censusRef(), scratchPath.resolve("census"), pr,
+                    bot.confOverrideRepository().orElse(null), bot.confOverrideName(), bot.confOverrideRef());
+        } catch (MissingJCheckConfException e) {
+            if (bot.confOverrideRepository().isEmpty()) {
+                log.info("No .jcheck/conf found in repo " + bot.repo().name() + ": " + e);
+                var text = " ⚠️ @" + pr.author().username() + " No `.jcheck/conf` found in the target branch of this pull request. "
+                        + "Until that is resolved, this pull request cannot be processed. Please notify the repository owner.";
+                addErrorComment(text, comments);
+            } else {
+                log.info("No .jcheck/conf found in external repo " + bot.confOverrideRepository().get().name() + ": " + e);
```

Comment:
The file isn't necessarily called `.jcheck/conf`, the name is given in `bot.confOverrideName()`.

The log should be at SEVERE level and the throwable should be included. To achieve this, you need to use the `log` method. By making it SEVERE, we will get admin notifications about this.

```suggestion
                log.log(Level.SEVERE, "Jcheck configuration file " + bot.confOverrideName() + " not found in external repo " + bot.confOverrideRepository().get().name(), e);
```